static/src/javascripts/components/enhancer/enhancer.js

```diff
@@ -1,8 +1,4 @@
-define([
-    'require'
-], function (
-    require
-) {
+define(function () {
```

Comment:
There are two ways of loading a module - via a dynamic require (`System.import` / `require([])`) or via a static require (`require('x') / `import {x} from 'x'`). A dynamic require has a callback, a static require is required statically before executing the module. It is only the static requires which we normalize during a build. The dynamic requires run through normalization each time.